### PR TITLE
Readme: Run tests locally

### PR DESCRIPTION
**Pull request by jablonskim**

### CODE REVIEW

**Comment by m-kostrzewa** [comment]
I'm wondering if we should also link to this doc in https://github.com/Juniper/contrail-windows-ci/blob/development/SELFCHECK.md ?

File: CIScripts/Test/Tests/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# How to run tests from local machine
+
+1. Deploy dev env using `deploy-dev-env` job.
+
+2. Install `powershel-yaml`:
```
Comment (m-kostrzewa) [inline CIScripts/Test/Tests/README.md]: typo

File: CIScripts/Test/Tests/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# How to run tests from local machine
+
+1. Deploy dev env using `deploy-dev-env` job.
```
Comment (m-kostrzewa) [inline CIScripts/Test/Tests/README.md]: Is there a doc we can link to here?

File: CIScripts/Test/Tests/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# How to run tests from local machine
```
Comment (m-kostrzewa) [inline CIScripts/Test/Tests/README.md]: we should specify that it's running tests against Juniper Windows CI lab.

File: CIScripts/Test/Tests/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+2. Install `powershel-yaml`:
+
+    ```Install-Module powershell-yaml```
+
+3. Copy `testenv-conf.yaml.sample` to `testenv-conf.yaml` file and replace all occurences of `<CONTROLLER_IP>`, `<TESTBED1_NAME>`, `<TESTBED1_IP>`, `<TESTBED2_NAME>`, `<TESTBED2_IP>` with proper values.
+
+4. Run selected test, e.g.:
+
+    ```Invoke-Pester -Script @{ Path = ".\TunnellingWithAgent.Tests.ps1"; Parameters = @{ TestenvConfFile = "testenv-conf.yaml"}; } -TestName 'Tunnelling with Agent tests'```
```
Comment (m-kostrzewa) [inline CIScripts/Test/Tests/README.md]: I wonder how to tie this document with https://github.com/Juniper/contrail-windows-ci/blob/development/SELFCHECK.md

File: CIScripts/Test/Tests/testenv-conf.yaml.sample [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  # Used by non-multitenancy tests
+  defaultProject: ci_tests
+
+testbeds:
+  - name: <TESTBED1_NAME>
+    address: <TESTBED1_IP>
+    username: Administrator
+    password: 'Contrail123!'
+  - name: <TESTBED2_NAME>
+    address: <TESTBED2_IP>
```
Comment (m-kostrzewa) [inline CIScripts/Test/Tests/testenv-conf.yaml.sample]: Maybe we should be more specific which IP from the 3 IPs it has should be entered here? (same for controller)

File: CIScripts/Test/Tests/README.md (hunk 2 above)
Comment (jablonskim) [reply CIScripts/Test/Tests/README.md]: As far as I know, currently there is no such doc.

File: CIScripts/Test/Tests/README.md [hunk 6]
```diff
@@ -1,13 +1,20 @@
-# How to run tests from local machine
+# How to run tests from local machine on VMs in Juniper Windows CI Lab
 
 1. Deploy dev env using `deploy-dev-env` job.
 
-2. Install `powershel-yaml`:
+2. Install Pester:
+
+    ```Install-Module -Name Pester -Force -SkipPublisherCheck```
```
Comment (m-kostrzewa) [inline CIScripts/Test/Tests/README.md]: I think it should be specifically Pester 4.2.0.
Pester 3.4.something is prebuilt into Windows AFAIK.

File: CIScripts/Test/Tests/README.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    * `<TESTBED1_NAME>`, `<TESTBED2_NAME>` - Testbeds hostnames
+    * `<TESTBED1_IP>`, `<TESTBED2_IP>` - Testbeds IP addresses, accessible from local machine (the same network as for Controller)
+
+5. Run selected test, e.g.:
+
+    ```Invoke-Pester -Script @{ Path = ".\TunnellingWithAgent.Tests.ps1"; Parameters = @{ TestenvConfFile = "testenv-conf.yaml"}; } -TestName 'Tunnelling with Agent tests'```
+
+## CI Selfcheck
+
+To run CI Selfcheck please see [this document](../../../SELFCHECK.md).
```
Comment (m-kostrzewa) [inline CIScripts/Test/Tests/README.md]: You misunderstand. Link to this document from selfcheck. Not link to selfcheck in this document :D

**Pull request merged**